docker/genomes-catalog-update/scripts/make_per_genome_annotations.py

```diff
@@ -33,21 +76,21 @@ def print_results(result_dict, header, outdir, tool):
             file_out.write('\n'.join(result_dict[acc]))
 
 
-def load_annotations(ann_file, ann_result):
+def load_annotations(ann_file, clusters):
+    ann_result = dict()
     header = ''
     with open(ann_file, 'r') as file_in:
         for line in file_in:
             line = line.strip()
-            if line.startswith('#'):
+            if line.startswith('#query'):
                 header = line
             else:
-                genome = line.split('_')[0]
-                if genome not in ann_result:
-                    logging.info('Genome {} is in the annotation file but not present in the representative genome '
-                                 'list'.format(genome))
-                else:
-                    ann_result[genome].append(line)
-    return header
+                rep_protein = line.split('\t')[0]
+                if rep_protein in clusters.keys():
```

Comment:
No need for the ".keys()" when checking if a item exists in a dict

Reply:
thank you! fixed in [5535af3](https://github.com/EBI-Metagenomics/genomes-pipeline/commit/5535af3650e252e63e61c49ec356441fbec5baf7)